root-sync/base/crossplane/rad-lab/gen-ai/composition-gen-ai.yaml

```diff
@@ -0,0 +1,205 @@
+---
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: rad-lab-gen-ai
+  annotations:
+    crossplane.io/composition-schema-aware-validation-mode: strict
+  labels:
+    provider: kubernetes
+    guide: self
```

Comment:
**Non-blocking question**: Could we remove these? 🤔

Reply:
Removed.